README.md

```diff
@@ -12,19 +12,19 @@ MLOS is a project to enable autotuning for systems.
 <!-- TOC -->
 
 - [MLOS](#mlos)
```

Comment:
This is auto-generated, please revert whitespace changes.
There's also a markdown lint job that runs to take care of these.